Tasks/Common/TlsHelper_/TlsHelper_.psm1

```diff
@@ -5,16 +5,20 @@ function Add-Tls12InSession {
     param()
 
     try {
-        if ([Net.ServicePointManager]::SecurityProtocol -notcontains 'Tls12') {
-            [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]3072
+        if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -notcontains 'Tls12') {
+            $securityProtocol=@()
+            $securityProtocol+=[Net.ServicePointManager]::SecurityProtocol
+            $securityProtocol+=[Net.SecurityProtocolType]3072
+            [Net.ServicePointManager]::SecurityProtocol=$securityProtocol
+            
             Write-Host (Get-VstsLocString -Key TLS12AddedInSession)
```

Comment:
Do we need to expose this as a user facing message ?

Reply:
Yes. We'll let the user know that we have added TLS 1.2 in session and that the network calls might use this protocol.